src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,47 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+
+public class Ui {
+    private Scanner sc = new Scanner(System.in);
+
+
+    public Ui() {
+
+    }
+
+    public void showWelcomeMessage() {
+        String output =
+                "Hello! I'm Duckymomo\n"
+                + "What can I do for you?";
+
+        formatAndPrint(output);
+    }
+
+    public String readInput() {
+        return sc.nextLine();
+    }
+
+    public void printOutput(String output) {
+        formatAndPrint(output);
+    }
+
+    /**
+     * Formats messages to be printed on the terminal
+     * @param input Message to be formatted
+     */
+
+    private void formatAndPrint(String input) {
+        StringBuilder output = new StringBuilder();
+        String[] split = input.split("\n");
+
+        System.out.println("    ____________________________________________________________");
```

Comment:
This is a magic string, since you are reusing it it might be better to declare it as a constant, and call it to improve readability.

Code Quality: Avoid Magic Literals